devscripts/run_tests.sh

```diff
@@ -11,4 +11,4 @@ else
     exit 1
 fi
 
-python3 -m pytest "$test_set"
+python3 -bb -Werror -m pytest "$test_set"
```

Comment:
do same in devscripts\run_tests.bat

Reply:
The reason I didn’t is that `run_tests.bat` invokes a `pytest` binary, which I assume takes the options meant for the module, not for the Python interpreter itself (those that come after `-m`).

I suppose I might try enabling the option with environment variables instead…

Reply:
You can replace it with `py -bb -Werror -m pytest`

Reply:
nvm, `py` may not always exist. Should be `python`. But then it can technically be python 2 :(

Reply:
Could have used `python3` inste—

> This PEP deliberately excludes any proposals relating to Microsoft Windows, as devising an equivalent solution for Windows was deemed too complex to handle here.

— <https://peps.python.org/pep-0394/#exclusion-of-ms-windows>

Nevermind.

Reply:
Could just use `py -3` for Windows.